<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Focus</class>
 <widget class="QDialog" name="X2Focus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>nStep Setup Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>440</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>426</width>
       <height>450</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>0</y>
        <width>415</width>
        <height>71</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Communication Settings</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Serial Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="deviceList"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Rescan</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>512</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>415</width>
        <height>311</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>415</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Advanced Settings</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Step Rate:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Wiring Phase:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Step Speed (ms):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>120</y>
         <width>62</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>25</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Temperature:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>25</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Not available</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>170</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_energized">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>401</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Keep Coils Energized After Move (Warning: motor will get hot!)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Step Mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_StepSpeed">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>254</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_StepRate">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>254</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_WirePhase">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>55</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Backlash Compensation*:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_Backlash">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <number>-500</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Reset">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>391</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Set current position to Zero (applies on OK)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Reverse">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>381</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Swap In/Out directions on motor</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>391</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>*Slack when direction changes.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>55</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>- for in/ + for out</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>247</x>
        <y>400</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>400</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
